js/renovation/ui/__tests__/scroll_view.test.tsx

```diff
@@ -81,7 +126,7 @@ describe('ScrollView', () => {
 
       describe('ScrollBy', () => {
         it('should scroll by positive distance as number in the vertical direction', () => {
-          const containerRefMock = { scrollTop: 150, scrollLeft: 0 };
+          const containerRefMock = createContainerRef({ top: 150, left: 0 } as Location);
```

Comment:
createContainerRef = (location: Partial<Location>)
And remove  as Location
const containerRefMock = createContainerRef({ top: 150, left: 0 });

Reply:
fix it